src/test/datascience/ipywidgets/cdnWidgetScriptSourceProvider.unit.test.ts

```diff
@@ -140,7 +134,7 @@ suite('DataScience - ipywidget - CDN', () => {
 
                 assert.deepEqual(value, { moduleName: 'HelloWorld' });
                 // Should not make any http calls.
-                verify(httpClient.exists(anything())).never();
+                // verify(httpClient.exists(anything())).never();
```

Comment:
Wanted to leave this commented, this way when the tests are fixed we add these checks back.
FYI - The tests are disabled and i dont want to fix the tests now (the'res already an issue for that)

Reply:
Do we need the issue linked here so we can associate the change needed when the issue is fixed?